pkgs/data/themes/levuaska-gtk-theme/default.nix

```
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, gtk_engines }:
+
+stdenv.mkDerivation {
+  version = "2021-1-31";
```

```suggestion
  version = "unstable-2021-01-31";
```
Since there are no releases/tags, this should be used instead, according to https://nixos.org/manual/nixpkgs/stable/#sec-package-naming
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in **"YYYY-MM-DD"** format. Also append **"unstable"** to the name - e.g., **"pkgname-unstable-2014-09-23"**.

---

I have amended the commit/PR to fix this. Thank you!